generate_client.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Generate golang client sdk from OpenAPI 3.0.0 spec
+
+
+function stage() {
```

Comment:
IMO it is more readable when top level calls are on the top of the file. Please create run() function and put [L53-EoF] there.

Reply:
Fixed in https://github.com/kentik/sla_dashboard_webapp/pull/2